virttest/utils_misc.py

```diff
@@ -440,14 +440,16 @@ def is_port_free(port, address):
     return free
 
 
-def find_free_port(start_port, end_port, address="localhost"):
+def find_free_port(start_port, end_port, address="localhost", reserved=[]):
```

Comment:
don't used empty list as default value.

Reply:
@xutian Updated default value to `None`, thanks.